castled-notifications/src/main/java/io/castled/android/notifications/CastledNotifications.kt

```
@@ -280,30 +280,39 @@ object CastledNotifications {
     @JvmStatic
     fun logInboxItemClicked(inboxItem: CastledInboxItem, buttonTitle: String) =
         castledScope.launch(Dispatchers.Default) {
-            AppInbox.reportEventWith(
-                inboxItem, buttonTitle, "CLICKED"
-            )
+            if (isInited() && getCastledConfigs().enableAppInbox)
+                AppInbox.reportEventWith(
+                    inboxItem, buttonTitle, "CLICKED"
+                )
         }
 
     @JvmStatic
     fun logInboxItemsRead(inboxItems: List<CastledInboxItem>) =
         castledScope.launch(Dispatchers.Default) {
-            AppInbox.reportReadEventsWithItems(inboxItems)
+            if (isInited() && getCastledConfigs().enableAppInbox)
```

better to log error than silently ignoring